### PR TITLE
[Integration tests]: Add Microsoft Access integration tests

### LINKED ISSUE #7978: [Integration Testing] Test Microsoft Acess Integration
The objective of this issue is to test the Microsoft Acess integration with MindsDB. Please test the integration locally via pip. 

## Resources

Here are some useful links:

1. Follow the [README.md](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/[access_handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/access_handler)/README.md) file. It stores all the commands that need to be tested.
2. Test Microsoft Acess  via  Pip installation. 

## Steps to Follow

1. Test the integration as outlined in the Resources above.
2. Save the test results, including commands and their outputs, in the `Microsoft Acess .md` file in a new GitHub repository in your account. [Here is a sample Manual_QA.md file.](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/mysql_handler/Manual_QA.md)
3. Create a PR in the [MindsDB repository](https://github.com/mindsdb/mindsdb) following these steps:
	- Go to the `/mindsdb/integrations/handlers/` folder and open the `Handlers_Manual_QA.md` file.
	- If all the tests passed, append this line: `Microsoft Acess integration Test Passed: <link to the test results in your GitHub repository>`
	- If some tests failed, create an issue to report it and append this line: `Microsoft Acess integration Test Failed: <link to the test results in your GitHub repository>, <link to the issue>`

## :loudspeaker: Additional rewards :trophy:

Each integration test brings 10+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left:

**Multiple people can work on this issue.**
**If you decide to work on this issue, comment below if you are going to test via pip or docker installation so that the other contributor can test with the other local installation**

### PR DESCRIPTION
**Pull request by adeyinkaezra123**
Fixes #7978

### Type of change
- [x] 📄 This change requires a documentation update
### Please describe what changes you made, in as much detail as possible
Tested the Microsoft Access Handler Manually with docker installation and reported findings.

### Results

- [x] There's a Bug 🪲 [Access Handler Not Behaving As Expected](https://github.com/mindsdb/mindsdb/issues/3947)

### CODE REVIEW

File: mindsdb/integrations/handlers/access_handler/Microsoft_Access.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Test the Microsoft Access data integration
+
```
Comment (chandrevdw31) [inline mindsdb/integrations/handlers/access_handler/Microsoft_Access.md:2]: You don't need to add this file to the handler

File: mindsdb/integrations/handlers/Handlers_Manual_QA.md [hunk 2]
```diff
@@ -30,3 +30,4 @@ Google Big Query Integration Test Passed: https://github.com/maulanaakbardj/mind
 </br>
 Web Crawler Integration Test Passed: https://github.com/Taheralicyclewala/MindsDB-Testing/blob/main/WebCrawler/Web_Crawler_QA.md
 </br>
+Microsoft Access Integration Test Failed: https://github.com/adeyinkaezra123/mindsdb/blob/access-integration-test/mindsdb/integrations/handlers/access_handler/Microsoft_Access.md
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/Handlers_Manual_QA.md:33]: This link goes to `404 - not found`. Please update it.

Comment (adeyinkaezra123) [reply mindsdb/integrations/handlers/Handlers_Manual_QA.md:33]: It was deleted at [#0269919](https://github.com/mindsdb/mindsdb/pull/7994/commits/026991960b6b78a5b608b468ee98a5fecb269e67)

Would update the Handlers manual to reference a separate repo

Comment (adeyinkaezra123) [reply mindsdb/integrations/handlers/Handlers_Manual_QA.md]: The link has been updated

**Comment by martyna-mindsdb** [comment]
@adeyinkaezra123 
Please check if the handler dependencies were imported successfully. Run this command: `show handlers where name='access'`

**Comment by martyna-mindsdb** [comment]
This should be the output:

image

**Comment by adeyinkaezra123** [comment]
Import success seems to be false
image.

image

**Comment by martyna-mindsdb** [comment]
@adeyinkaezra123 
You need to install the dependencies and test it again.

**Comment by chandrevdw31** [comment]
@adeyinkaezra123 the issue advises to test with pip, not docker

**Comment by chandrevdw31** [comment]
user resubmitted as #8012

**Pull request closed without merging**